### PR TITLE
ci: created Dockerfile and docker.yml for building and pushing docker…

### PR DESCRIPTION
**Pull request by thomkainos**
### Describe your changes
This pr creates contains the specs required to build a Docker image for our Job Role Manager System. It also includes an addition step in our CI pipeline to configure aws credentials, login to aws ecr, and then build and push the latter image. 

## Issue ticket number and link
US036

https://tasks.office.com/kainos.com/Home/Task/v1gvHMIftkymdgvXcvfU0JYAH2Q-?Type=TaskLink&Channel=Link&CreatedTime=638580286455020000

#### General
- [x] I have performed a self-review of my code
- [x] Code changes are considered complete for the given story (i.e., the comments from code review are addressed)
- [x] The commit title follows conventional commit standards
- [x] I have put the PR link into the planner
- [x] The PR has a clear and descriptive title and a detailed description of the changes
- [x] Code builds cleanly without any errors or warnings

#### Code Quality
- [x] I have run linting and there were no errors

#### Testing
- [x] New functionality is tested
- [x] All unit and integration tests passed

#### Deployment
- [x] Documentation has been updated where needed
- [x] Any required database migrations have been added and tested

#### Security
- [x] No sensitive data is hard-coded or exposed

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ENV DB_HOST ${DB_HOST}
+ENV DB_PASSWORD ${DB_PASSWORD}
+ENV DB_USERNAME ${DB_USERNAME}
+ENV DB_NAME ${DB_NAME}
+
+RUN mvn clean install -DskipTests=true
+
+EXPOSE 8080
+
+CMD ["java","-jar", "/code/target/java-dropwizard-flyway-starter-1.0-SNAPSHOT.jar", "server", "/code/config.yml"]
```
Comment (swalker125) [inline Dockerfile]: is this correct? we renamed our project to kainos-job-role-manager, should we be expecting to use the old project name for the jar file?

Comment (thomkainos) [reply Dockerfile]: No we shouldn't, just changed to reflect new name

Comment (swalker125) [reply Dockerfile]: Search in the code for instances of java-dropwizard-flyway-starter (shift shift)

Comment (thomkainos) [reply Dockerfile]: Just updated

**Review by erinmcj — DISMISSED** [review]

File: README.md
Comment (swalker125) [inline README.md:1]: surprised to see all these changes here - wasn't this done in the update docs ticket?

**Review by swalker125 — APPROVED** [review]
LGTM

**Review by erinmcj — APPROVED** [review]

**Pull request merged**